packages/react-urql/examples/1-getting-started/public/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="charset" value="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>urql Example: 1. Getting Started</title>
+
+    <script>
+      navigator.serviceWorker.getRegistrations().then(registrations => {
```

Comment:
TIL, did not know about this

Reply:
It seems to be the only way to forcefully delete CodeSandbox's service worker.